game/plugin/skills/fishing/src/org/apollo/game/plugin/skills/fishing/FishingTarget.kt

```diff
@@ -0,0 +1,39 @@
+package org.apollo.game.plugin.skills.fishing
+
+import org.apollo.game.model.Position
+import org.apollo.game.model.entity.Player
+import org.apollo.game.plugin.api.fishing
+import org.apollo.game.plugin.api.rand
+import org.apollo.game.plugin.skills.fishing.FishingAction.Companion.hasBait
+import org.apollo.game.plugin.skills.fishing.FishingAction.Companion.hasTool
+
+data class FishingTarget(val position: Position, val option: FishingSpot.Option) {
+
+    /**
+     * Returns whether or not the catch was successful.
+     * TODO: We need to identify the correct algorithm for this
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([kotlin:S1135](https://sonarcloud.io/organizations/github-apollo-rsps/rules#rule_key=kotlin%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=apollo%3Aorg.apollo&pullRequest=338&issues=AWVzgXxBXKENQn1Ssjbl&open=AWVzgXxBXKENQn1Ssjbl)

[ISSUE_KEY]: AWVzgXxBXKENQn1Ssjbl